crest/Assets/Crest/Crest/Scripts/OceanRenderer.cs

```diff
@@ -711,6 +721,42 @@ void LateUpdateLods()
             _lodDataShadow?.UpdateLodData();
         }
 
+        void LateUpdateTiles()
+        {
+            // If there are local bodies of water, this will do overlap tests between the ocean tiles
+            // and the water bodies and turn off any that don't overlap.
+            if (_waterBodies.Count == 0) return;
```

Comment:
We want to re-enable all the ocean chunk renderers in this case - default back to a full ocean. Otherwise you just get all the previously enabled tiles just drifting around.

Reply:
Yeah its a good point.. i guess it probably wants state so that it doesnt need to loop over all the renderers each frame? i can look at that

Reply:
i put in a flag that defers this early-out until after a final culling pass has been performed. so if the number of water bodies is 0, it does a final culling pass (which enables everything), then sets the flag true.

this seemed to be the safest way to do it. let me know if this works